{{cookiecutter.github_repository}}/provisioner/roles/celery/templates/celerybeat.service.j2

```diff
@@ -9,10 +9,7 @@ Group={{ celery_group }}
 Type=forking
 Restart=always
 WorkingDirectory={{ project_path }}
-ExecStart={{ venv_path }}/bin/celery beat start worker-beat-{{ project_namespace }} -A {{ project_name }}  -l {{ celery_log_level }} \
-    --logfile={{ celerybeat_log_file }} --pidfile={{ celerybeat_pid_file }}  --schedule={{ celerybeat_schedule_file}}
-ExecStop={{ venv_path }}/bin/celery beat stop worker-beat-{{ project_namespace }} --pidfile={{ celerybeat_pid_file }}
-ExecReload={{ venv_path }}/bin/celery beat restart worker-beat-{{ project_namespace }} -A {{ project_name }} -l {{ celery_log_level }} \
+ExecStart={{ venv_path }}/bin/celery beat -A {{ project_name }}  -l {{ celery_log_level }} \
```

Comment:
Since you are removing the ExecStop, does `ExecStart` takes care of things when one writes `server <process-name> stop`?

Reply:
great